test/development/acceptance/hydration-error.test.ts

```diff
@@ -1277,6 +1277,7 @@ describe('Error overlay for hydration errors in Pages router', () => {
                           <Mismatch>
                             <p>
                               <span>
+                                ...
```

Comment:
Sweet. Looks like this matches the Turbopack snapshot now which makes me wonder how this  ever worked for Turbopack but not Webpack?